web_hosting_manager/README.md

```diff
@@ -3,7 +3,7 @@ The tutorial app show cases how to create and manage web services for Public ID
 
 ## Install
 
-* **Note: requires a node version 8.0.0 and an npm version 5.0.0**
+* **Note: requires a node version ^8.0.0 and an npm version ^5.0.0**
```

Comment:
Perhaps we want to enforce this from the `package.json` as we do it in the email app to prevent people using previous versions and running into unexpected errors caused by some unknown incompatibility issues among modules.

Reply:
A `devEngines` property already exists in `web_hosting_manager/package.json` enforcing node and npm versions.

Reply:
Hi, I was trying to install all of the safe examples but I have npm version 6.4.1 so an AssertionError was raised expecting my npm version to satisfy 3.x || 4.x || 5.x. I could downgrade my npm but I was wondering why later versions of npm aren't supported.
Thanks

Reply:
Hi @Charles-Johnson , it should also work with that version of npm, in fact it works for me. Note we just migrate this project to its own repository, can you please clone this other repo instead and try to build it from there? https://github.com/maidsafe/safe-web-hosting-manager-electron